### PR TITLE
Darkabsolute adding information to file

### LINKED ISSUE #21: In my personal file adding my favorites games and anime, etc.
In this file adding my favorites games, anime and manga, but it's my personal opinion.

### PR DESCRIPTION
**Pull request by DarkAbsolutE**
Adding my favorites games, animes and mangas.
Adding a programming language.

This work it's finish closes #21

### CODE REVIEW

File: darkabsolute.md [hunk 1]
```diff
@@ -1,4 +1,21 @@
 - My favorites lenguages of programming-
 - Android :D
 - Javascript
-- etc... 
+- etc...
+
+- My favorites games:
+
+
+- Bioshock 1 and 2
```
Comment (FerPerales) [inline darkabsolute.md:9]: Bioshock rules :+1:

**Comment by FerPerales** [comment]
Looks good to me

File: darkabsolute.md [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+- Final Fantasy XIII 1 and 2
```
Comment (githubteacher) [inline darkabsolute.md:10]: FF7 is getting remade too!

File: darkabsolute.md [hunk 3]
```diff
@@ -1,4 +1,21 @@
...
+- Hyperdimention neptunia 1 and 2
+- Ori
+- etc...
+
+- My favorites animes and mangas:
+- Fullmetal alchemist brotherhood
```
Comment (githubteacher) [inline darkabsolute.md:16]: An important difference in the name of this anime! :wink:

**Pull request merged**